source/likely.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
 var likely = require('./index.js');
+var history = require('./history');
+var utils = require('./utils');
 
 window.likely = likely;
-window.addEventListener('load', likely.initiate);
+window.addEventListener('load', function () {
+    if (likely.defaults.listenChangesUrl) {
+        history.onChangesUrl = function () {
```

Comment:
`onChangesUrl` is a noop by default. If a user decided to listen changes urls (which is turned on, by default) then likely starts to reinit when it happens.